framework/src/imgui_manager.cpp

```diff
@@ -176,6 +155,14 @@ namespace gvk
         io.DisplayFramebufferScale = ImVec2(1.0f, 1.0f); // TODO: If the framebuffer has a different resolution as the window
 	    io.DeltaTime = gvk::time().delta_time();
 
+
+		// new frame and callbacks have to be here, to give the updater an opportunity to clean up (callbacks may cause update events)
+		ImGui_ImplVulkan_NewFrame();
+		ImGui::NewFrame();
```

Comment:
I'm not sure if it is related to these calls... but might be (not an ImGui-expert) ^^

The following assert fails in `imgui.cpp`, line #6744:
```
// Verify that io.KeyXXX fields haven't been tampered with. Key mods shoudl not be modified between NewFrame() and EndFrame()
const ImGuiKeyModFlags expected_key_mod_flags = GetMergedKeyModFlags();
IM_ASSERT(g.IO.KeyMods == expected_key_mod_flags && "Mismatching io.KeyCtrl/io.KeyShift/io.KeyAlt/io.KeySuper vs io.KeyMods");
```
Mind the comment! My guess is that there is some issue with that.

It happens in debug mode when I press the _Shift_ key.

Reply:
The relevant part of code is reorganized.